cluster/examples/gitlab/gcp/resource-claims/buckets/artifacts.yaml

```diff
@@ -0,0 +1,11 @@
+# Example GCP Bucket resource claim using the gcp-bucket resource class
+apiVersion: storage.crossplane.io/v1alpha1
+kind: Bucket
+metadata:
+  name: gitlab-artifacts
+  namespace: default
+spec:
+  classReference:
+    name: standard-gcp-bucket
+    namespace: crossplane-system
+  name: gitlab-demo-artifacts
```

Comment:
I wonder whether we should somehow handle the fact that these names are globally unique - add a prefix via a script or something.

Reply:
I think that is a great idea for follow up PR.

Reply:
https://github.com/crossplaneio/crossplane/issues/431